src/notifications/screens/notifications.screen.js

```diff
@@ -145,6 +148,7 @@ class Notifications extends Component {
     this.props.getUnreadNotifications();
```

Comment:
Would prefer is we destructure here as well (I know this is slightly out of your scope however so don't worry if you would rather not):

```
     const {
       getUnreadNotifications,
       getParticipatingNotifications,
       getAllNotifications,
       getNotificationsCount
      } = this.props;
```

And then `Promise.all` all the functions instead of calling them one by one

Reply:
Done except promises, how can they help?

Reply:
No worries, ideally `Promise.all` is used to make sure a _collection_ of promises are complete before submitting an event after them but since we're not really doing anything after each of these specifically - I think it's okay if we don't include it here :)